Docker/debian/files/sources.list

```diff
@@ -0,0 +1,8 @@
+deb http://security.debian.org/ jessie/updates main
```

Comment:
This doesn't appear to be used anywhere.

Reply:
Fixed.